### ISSUE TITLE
CLI - Add topic browsing commands

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#172]
The CLI should allow listing topics and contained collection and data records.

**Comment by JoshuaSBrown** [comment]
What are we calling topics? Are these the same as tags?

**Comment by dvstans** [comment]
No, topics are how "published" data is organized in the datafed catalog.
